### ISSUE TITLE
[bug] dask_gateway image/Dockerfile not included during initial qhub deploy

### ISSUE THREAD

**Issue by iameskild** [Quansight/qhub#568]
## Describe the bug
While deploying datum's new QHub on AWS, the first initial init and deploy ran successfully. However, the first push to Github resulted in a failed Docker Image Build workflow. The error messages are below, but here are the qhub commands:

`qhub init aws --project datum-prod --domain datum.qhub.dev --ci-provider github-actions --auth-provider auth0 --auth-auto-provision --repository github.com/datum-tech/qhub-datum-aws-prod --repository-auto-provision`
* domain has since been updated to `qhub.datumxy.com`

`qhub deploy -c qhub-config.yaml --dns-auto-provision `
* DNS registration was handled by a datum employee.

From reading the documentation, specifically the section on [default images](https://docs.qhub.dev/en/latest/source/02_get_started/04_configuration.html#default-images) and comparing it to what was present in the qhub-config.yaml, it appeared that the `dask_gateway: <default image>` was missing. The `Dockerfile.dask-gateway` also appeared to be missing. 

Once these were added, the Docker Image Build didn't restart but the other workflows ran successfully. And after a few minor, unrelated tweaks, the Qhub appears to be up and running. 

## How can we help?
Help us help you.
* What are you trying to achieve?
    * Reporting this bug for visibility. 
    * These default images still have not been pushed up to the ECR registry.
* How can we reproduce the problem?
    * Perhaps by trying to deploy a fresh QHub on AWS 
* What is the expected behaviour?
* And what is currently happening?
* Any error messages?
    * These error messages are from the failed Docker Image Build GitHub workflow but seem somewhat unrelated to the issue described above:
```
name unknown: The repository with name 'datum-prod-dask-worker' does not exist in the registry with id '***'
Error: Process completed with exit code 1.
```  
> If helpful, please add any screenshots and relevant links to the description.

## Your environment
Describe the environment in which you are experiencing the bug.
> Include your conda version (use `conda --version`), k8s and any other relevant details.

```
conda 4.10.1
qhub version 0.3.7
# k8s
Client Version: v1.20.2
Server Version: v1.17.17-gke.3000
```

**Comment by costrouc** [comment]
@iameskild could you try using qhub version 0.3.10? We had to build our own version of dask gateway (which is currently a small PR at dask-gateway waiting to be merged). There are also docs on upgrades https://docs.qhub.dev/en/latest/source/02_get_started/08_upgrade.html. There is a new dask_gateway key detailed in the docs https://docs.qhub.dev/en/latest/source/02_get_started/04_configuration.html#default-images.

**Comment by iameskild** [comment]
Thanks @costrouc! I have gone ahead and upgraded QHub on our CI (and also upgrade the dask/jupyter related) images. 
After talking to @brl0, this issue might be conflating two different things. One of which has now been resolved with the 0.3.11 upgrade. The other issue we're

**Comment by iameskild** [comment]
Issue resolved with 0.3.11 deployment.
